netty/src/main/java/io/grpc/transport/netty/NettyClientTransport.java

```diff
@@ -192,8 +192,10 @@ public void shutdown() {
   }
 
   private void notifyShutdown(Throwable t) {
+    Status s = Status.OK.withDescription("Transport is shutting down");
     if (t != null) {
       log.log(Level.SEVERE, "Transport failed", t);
+      s = s.withCause(t).withDescription("Transport failed");
```

Comment:
But now this line doesn't cause the ChannelImpl to be informed it needs to do backoff. If `t` is `null` we do want OK, but if `t` is not `null` we do need a non-OK status.